.pycodestyle

```diff
@@ -0,0 +1,6 @@
+[pycodestyle]
+count = False
+ignore = E226,E401,E402,W504
```

Comment:
Any reason why we are ignoring these four things?

E226 - Whitespace around operator
E401 - Multiple imports on one line
E402 - Import not at top of file
W504 - Line break after binary operator instead of before

Reply:
fixed.
@CoryMartin-NOAA 
The exgdas scripts are loading `ufsda` and not at the top of the code.  For now we are retaining `E402`, but I suggest we remove that soon.

Reply:
I hope to remove that sooner rather than later and wrap it into `pygw`.

Reply:
E402 and W504 deferred to the future.